bindings/java/src/test/com/apple/foundationdb/test/RangeTest.java

```diff
@@ -34,11 +35,10 @@
 import com.apple.foundationdb.async.AsyncIterable;
 
 public class RangeTest {
-	private static final int API_VERSION = 720;
 
 	public static void main(String[] args) {
-		System.out.println("About to use version " + API_VERSION);
-		FDB fdb = FDB.selectAPIVersion(API_VERSION);
+		System.out.println("About to use version " + ApiVersion.LATEST);
+		FDB fdb = FDB.selectAPIVersion(ApiVersion.LATEST);
```

Comment:
Doing this introduces a responsibility to keep the code up to date with API changes. Are we regularly running these tests now so that we would know if they broke due to API changes?

Reply:
These tests in particular are not executed a part of the CI, but we update these versions mechanically on every version bump anyway.

Reply:
I don't think we historically bumped these versions on every release, except in cases where we evaluated the API changes against the tests. If that's happening now, I think it's a function of this process being somewhat difficult to judge when to update. Still, usually we wouldn't make code run on a dynamic API version without evaluating whether it actually works on the updated API version.

Reply:
We are executing Java unit tests, Java integration tests & binding tester as part of our CI. Unfortunately, we are not executing the tests java/src/test, which means they could be broken for a while already without us knowing that. If these tests are useful we should call them from CI. Of course, I can also revert these tests to hardcoded API versions.

Reply:
My gut says we should use a hard-coded API version (maybe adding a comment to explain why) unless we start regularly running the tests. The hard-coded version can be updated any time someone wants to run the test against a new version by checking the list of API changes to see if anything needs to be updated.

That said, I don't feel super strongly about this since it's not likely to affect anybody given that we don't run them. If we leave it as written here, it means that someone calling the test may have to debug API version related problems if they want to dust it off and try to use it.

Reply:
I have introduced a separate API type declaration for the standalone Java tests & benchmarks that are not running as a part of the CI. It is in TestApiVersion.java